corehq/apps/translations/integrations/transifex/views.py

```diff
@@ -31,21 +22,32 @@
     AddTransifexBlacklistForm,
     AppTranslationsForm,
     ConvertTranslationsForm,
+    DownloadAppTranslationsForm,
     PullResourceForm,
 )
-from corehq.apps.translations.generators import Translation, PoFileGenerator
-from corehq.apps.translations.integrations.transifex.exceptions import ResourceMissing
+from corehq.apps.translations.generators import PoFileGenerator, Translation
+from corehq.apps.translations.integrations.transifex.exceptions import (
+    ResourceMissing,
+)
+from corehq.apps.translations.integrations.transifex.transifex import Transifex
+from corehq.apps.translations.integrations.transifex.utils import (
+    transifex_details_available_for_domain,
+)
 from corehq.apps.translations.models import TransifexBlacklist
 from corehq.apps.translations.tasks import (
-    push_translation_files_to_transifex,
-    pull_translation_files_from_transifex,
-    delete_resources_on_transifex,
     backup_project_from_transifex,
+    delete_resources_on_transifex,
+    download_project_from_hq,
+    pull_translation_files_from_transifex,
+    push_translation_files_to_transifex,
```

Comment:
is the order relevant? just curious why it changed?

Reply:
(ditto [isort](https://isort.readthedocs.io/en/stable/))

Reply:
looks like it aligned them alphabetically. 👍